docs/INSTALL_CONVENTIONS.md

```diff
@@ -0,0 +1,147 @@
+# Installation Script Conventions
+
+## File Structure
+All installation scripts should follow this standard structure:
+
+### 1. File Header
+```bash
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 community-scripts ORG
+# Author: tteck (tteckster)
+# License: MIT
+# https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+```
+
+### 2. Initial Setup
+Every script should source the functions file and run these initial checks:
+
+```bash
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+```
+
+### 3. Standard Dependencies
+Common base dependencies should be installed first:
+
+```bash
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
```

Comment:
```suggestion
$STD apt-get install -y \
  curl \
  sudo \
  mc 
```
Is this not the standard? At least that is what @MickLesk suggested on one of my PR